src/common/output/check.rs

```diff
@@ -16,6 +16,17 @@ impl CheckOutput {
         }
     }
 
+    /// Prints information when there is nothing to check and returns a boolean
+    pub fn is_something_to_check(&self) -> bool {
+        if self.files_count == 0 {
+            if !self.is_quiet_mode {
+                println!("Nothing to check");
+            }
+            return false;
+        }
+        true
+    }
```

Comment:
```suggestion
    pub fn print_nothing_to_check(&self) {
        if !self.is_quiet_mode && self.files_count == 0 {
            println!("Nothing to check");
        }
    }
```
This method should print a message and not return anything.

Reply:
I used the return value of this function to return early in the check function [here](https://github.com/dotenv-linter/dotenv-linter/pull/392/files#diff-b1a35a68f14e696205874893c07fd24fdb88882b47c23cc0e0c80a30c7d53759R20).

This way I could use the function to notice me about an empty input list without calling the `is_empty()` method again.

Reply:
> I used the return value of this function to return early in the check function [here](https://github.com/dotenv-linter/dotenv-linter/pull/392/files#diff-b1a35a68f14e696205874893c07fd24fdb88882b47c23cc0e0c80a30c7d53759R20).
> 
> 
> 
> This way I could use the function to notice me about an empty input list without calling the `lines_map.is_empty()` method again.

Personally, I think this is not very clear logic. Because there is an implicit logic inside this method - output to the console, which is not clear from the name (`is_something_to_check`).  
In addition, I believe that this is not `Output`'s area of ​​responsibility.

Reply:
You are right it's not very clear.

I will change it :)

Reply:
I moved the `lines_map.is_empty()` check outside of the `Output` struct.
Now the `print_nothing_to_check()` function just prints the output.